build-parent/pom.xml

```diff
@@ -1032,6 +1035,24 @@ To suppress enforce-helm-version rule, you must add following plugin to the root
                 <hide.manual.actions>true</hide.manual.actions>
             </properties>
         </profile>
+        <profile>
```

Comment:
I: legacy aissemble build workaround to prevent duplicate pushing of the same `.dev.<timestamp>` version into test.pypi.org.  While some repositories (e.g., nexus) allow this, test.pypi.org does not.